dist/tools/testrunner/testrunner.py

```diff
@@ -37,7 +37,10 @@ def run(testfunc, timeout=10, echo=True, traceback=False):
         return 1
     finally:
         print("")
-        os.killpg(os.getpgid(child.pid), signal.SIGKILL)
-        child.close()
+        try:
+            os.killpg(os.getpgid(child.pid), signal.SIGKILL)
+        except ProcessLookupError:
+            print("Process lookup failed")
 
+        child.close()
```

Comment:
This line could be put in a `finally` block:
```python
try:
    os.killpg(os.getpgid(child.pid), signal.SIGKILL)
except ProcessLookupError:
    print("Process lookup failed")
finally:
    child.close()
```

Reply:
It's already in a `finally` (outer) block.